Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -47,12 +47,56 @@
     return s_dwriteFactory;
 }
 
+template <typename TElement>
+bool __CloneArray(_In_reads_opt_(count) TElement const* source,
```

Comment:
raw pointers are valid iterators for stl

---
In reply to: [78621358](https://github.com/Microsoft/WinObjC/pull/999#discussion_r78621358) [](ancestors = 78621358,78468942)